test/scales/linearScaleTests.ts

```diff
@@ -188,9 +187,9 @@ describe("Scales", () => {
     });
 
     describe("autoranging behavior", () => {
-      var data: any[];
-      var dataset: Plottable.Dataset;
-      var scale: Plottable.Scales.Linear;
+      let data: any[];
+      let dataset: Plottable.Dataset;
+      let scale: Plottable.Scales.Linear;
```

Comment:
Why is this scale a `let` and not a `const`? I just feel like this `let` / `const` will lead to inconsistencies